bundles/org.openhab.binding.haywardomnilogic/src/main/java/org/openhab/binding/haywardomnilogic/internal/HaywardHandlerFactory.java

```diff
@@ -96,11 +99,12 @@ public HaywardHandlerFactory(@Reference HttpClientFactory httpClientFactory) {
         if (thingTypeUID.equals(HaywardBindingConstants.THING_TYPE_HEATER)) {
             return new HaywardHeaterHandler(thing);
         }
+        if (thingTypeUID.equals(HaywardBindingConstants.THING_TYPE_PUMP)) {
+            return new HaywardPumpHandler(thing);
+        }
         if (thingTypeUID.equals(HaywardBindingConstants.THING_TYPE_RELAY)) {
             return new HaywardRelayHandler(thing);
-        }
-        if (thingTypeUID.equals(HaywardBindingConstants.THING_TYPE_SENSOR)) {
-            return new HaywardSensorHandler(thing);
```

Comment:
This should be mentioned in the update messages to be not backward compatible.

Reply:
Is there any documentation available on the update.lst syntax?